site/docs/5_references/1_nifi_cluster/7_external_service_config.md

```diff
@@ -14,6 +14,30 @@ ListenersConfig defines the Nifi listener types :
         portConfigs:
           - port: 8080
             internalListenerName: "http"
+          - port: 7182
+            internalListenerName: "my-custom-listener"
+            protocol: TCP
+      metadata:
+        annotations:
+          toto: tata
+        labels:
+          titi: tutu
+```
+
+Load balancer example :
```

Comment:
```suggestion
Load balancer example:
```

And can you do it too on `ListenersConfig defines the Nifi listener types :`?

Reply:
Just some typo 😄

Reply:
Done